apps/core-app/src/components/MemberProfileMenu.tsx

```diff
@@ -0,0 +1,144 @@
+import { useConnectedMembership } from '@daohaus/dao-context';
+import {
+  Dropdown,
+  DropdownMenuItem,
+  DropdownButton,
+  font,
+  Theme,
+  Dialog,
+  DialogTrigger,
+  DialogContent,
+  Button,
+  DropdownLink,
+  DropdownText,
+} from '@daohaus/ui';
+import { useMemo } from 'react';
+import { BiDotsVerticalRounded } from 'react-icons/bi';
```

Comment:
Is there a Remix Icon for something similar? 
Sounds like that is the icon pack we're trying to use exclusively

Reply:
found it